install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -73,6 +73,16 @@ type OauthConfiguration struct {
 	SarNamespace    string `json:"sarNamespace,omitempty"`
 }
 
+type PublicApiConfiguration struct {
+	Enabled bool `json:"enabled,omitempty"`
+	// Set RouteHostname to the hostname of the exposed syndesis API.  Typically the operator can automatically
+	// determine this by looking at the result of the Route object it creates.
+	RouteHostname string `json:"routeHostname,omitempty"`
+	// if set to true, then any authenticated user can access the API. otherwise the user
+	// needs access to get pods against the SarNamespace
+	DisableSarCheck bool `json:"disable-sar-check,omitempty"`
+}
```

Comment:
I would like to see the default values for these fields in 
https://github.com/syndesisio/syndesis/blob/684df965f085499789b8033385abb63027e4b255/install/operator/build/conf/config.yaml#L1-L78

Reply:
Added default values in config.yaml.